src/dsp_tools/utils/connection_live.py

```diff
@@ -294,7 +295,10 @@ def _handle_non_ok_responses(self, response: Response, request_url: str, retry_c
             self._log_and_sleep("Transient Error", retry_counter, exc_info=False)
             return None
         else:
-            msg = f"Permanently unable to execute the network action. See logs for more details: {LOGFILES}"
+            msg = "Permanently unable to execute the network action. "
+            if original_str := regex.search(r'{"knora-api:error":"dsp\.errors\.(.*)","@context', str(response.content)):
+                msg += f"\n{' '*37}Original Message: {original_str.group(1)}\n"
```

Comment:
Noemi came to me with this bug. She told me that it would be helpful to her if this message is not only found in the logs.